resources/function_help/json/shared_paths

```diff
@@ -0,0 +1,23 @@
+{
+  "name": "shared_paths",
+  "type": "function",
+  "groups": ["GeometryGroup"],
+  "description": "Returns a collection containing paths shared by the two input geometries. Those going in the same direction are in the first element of the collection, those going in the opposite direction are in the second element. The paths themselves are given in the direction of the first geometry.",
```

Comment:
It would be nice to have the detail of what is return and how it's selected. In this case it seems to return nearby vertexes that are in the same sequence and make  a line with them. So it uses those vertex to make a new line and doesn't return part of the input geometry. Or maybe I'm interpreting the examples wrong.

Reply:
The postgis st_sharedpaths docs have some visual examples which may help: https://postgis.net/docs/ST_SharedPaths.html

Reply:
Could a mention or a reference to this be added?

Reply:
I'm not super keen to link to external resources here. I'd rather just wait till we have capacity to insert images in these docs...

Reply:
The image is st_sharedpath is somewhat helpful, though it still not clear if lines without a shared vertex will be supported or the 'paths' are only when vertices match for both lines? Such as this case or something similar 
![image](https://user-images.githubusercontent.com/12854129/188332603-3598a1b8-d3b6-4b1a-bd5f-3135b2bc02a0.png)